testprojects/src/python/python_distribution/ctypes/setup.py

```diff
@@ -7,10 +7,9 @@
 
 from setuptools import setup, find_packages
 
-
 setup(
   name='ctypes_test',
-  version='0.0.1',
+  version='1.0.0',
```

Comment:
1.0.0 typically means stable, so this may not be yet? (your call)
A good reference would be https://github.com/pantsbuild/pants/blob/master/src/python/pants/backend/project_info/tasks/export.py#L51-L63

so maybe `0.1.0` if there is backward incompatibility?

Reply:
Hm, I hadn't thought about the significance of this at all, mostly changed it to mimic the other `python_dist()` targets. I don't think it's stable yet, I was planning to switch the target names from `ctypes_compatible_cpp_library()` to just `cpp_library()` at that time as well. I'll revert this change.

Reply:
Thanks a lot for the reference!

Reply:
There is backwards incompatibility, but I don't think anyone would be relying on the broken behavior this PR fixes, except accidentally, so I think `0.0.1` is probably the right move here.

Reply:
Reverted the version change in f88e427!